Flow.Launcher/Helper/SingleInstance.cs

```diff
@@ -12,375 +12,372 @@
+            // shared location
+            string appFolderPath = Path.Combine(
+                Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), uniqueApplicationName);
+
+            string cmdLinePath = Path.Combine(appFolderPath, "cmdline.txt");
+            if (File.Exists(cmdLinePath))
             {
-              
-                // The application was clickonce deployed
-                // Clickonce deployed apps cannot recieve traditional commandline arguments
-                // As a workaround commandline arguments can be written to a shared location before 
-                // the app is launched and the app can obtain its commandline arguments from the 
-                // shared location               
-                string appFolderPath = Path.Combine(
-                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), uniqueApplicationName);
-
-                string cmdLinePath = Path.Combine(appFolderPath, "cmdline.txt");
-                if (File.Exists(cmdLinePath))
+                try
+                {
+                    using TextReader reader = new StreamReader(cmdLinePath, Encoding.Unicode);
+                    args = NativeMethods.CommandLineToArgvW(reader.ReadToEnd());
+
+                    File.Delete(cmdLinePath);
+                }
+                catch (IOException)
                 {
-                    try
-                    {
-                        using (TextReader reader = new StreamReader(cmdLinePath, Encoding.Unicode))
-                        {
-                            args = NativeMethods.CommandLineToArgvW(reader.ReadToEnd());
-                        }
-
-                        File.Delete(cmdLinePath);
-                    }
-                    catch (IOException)
-                    {
-                    }
                 }
             }
+        }
 
-            if (args == null)
-            {
+        if (args == null)
+        {
                 args = new string[] { };
-            }
-
-            return new List<string>(args);
         }
 
-        /// <summary>
-        /// Creates a remote server pipe for communication. 
-        /// Once receives signal from client, will activate first instance.
-        /// </summary>
-        /// <param name="channelName">Application's IPC channel name.</param>
-        private static async Task CreateRemoteService(string channelName)
+        return new List<string>(args);
+    }
+
+    /// <summary>
+    /// Creates a remote server pipe for communication.
+    /// Once receives signal from client, will activate first instance.
+    /// </summary>
+    /// <param name="channelName">Application's IPC channel name.</param>
+    private static async Task CreateRemoteService(string channelName)
+    {
+        using (NamedPipeServerStream pipeServer = new NamedPipeServerStream(channelName, PipeDirection.In))
```

Comment:
Didn't touch this `using` block and the next one because they're doing async stuff, and I didn't want to accidentally break something.

Reply:
I think you can change it to be await using...but not a big deal.

Reply:
Maybe could you revert back all the format change in `SingleInstance.cs`? As #2616 is also changing this file (massively), it might be annoying to merge the format change....

Reply:
Done.